src/xml2js_output_parser.ts

```diff
@@ -41,4 +41,15 @@ export default class Xml2jsOutputParser<T> {
     }
     return parsed;
   }
+
+  parseSubtag<U>(tag: XMLInterface.XMLTagWithSubtags, subtagName: string, subtagParser: Xml2jsOutputParser<U>): U {
+    let subtags = tag[subtagName];
```

Comment:
this could call parseSubtags and return the first result, it's just about the same